composer.json

```
@@ -15,37 +15,37 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "php": "^8.0 || ^8.1",
+    "php": "^8.1",
```

See comments in https://github.com/pimcore/pimcore/pull/14080
```suggestion
    "php": "~8.1.0 || ~8.2.0",
```

---

Done in [f3c4204](https://github.com/pimcore/customer-data-framework/pull/416/commits/f3c42045784f6d9b1d9c20fa1590386408d310ce)